src/assets/_project/_blocks/qg-main.js

```diff
@@ -49,6 +52,8 @@ import './utils/qg-init';
   shareLinks.init();
   accessibility.init();
 
-  $('.qg-index-links .qg-index-item img').length === 0 ? $('.qg-index-links').addClass('content-only') : '';
+  if ($('.status').length > 0) {
```

Comment:
If this needs to be revisited while getting rid of status class, please mention details in comments //TODO: